app/src/models/Inventory.ts

```diff
@@ -163,6 +166,11 @@ export class Inventory
           allowNull: true,
           field: "global_warming_potential_type",
         },
+        lastUpdatedAt: {
+          type: DataTypes.DATE,
+          allowNull: true,
+          field: "last_updated_at",
+        },
       },
       {
         sequelize,
```

Comment:
We have the `last_updated` timestamp defined below and we can manually overwrite it in other queries/ endpoints if necessary. Why do we need 2 last update timestamp fields?